Library/Formula/cgdb.rb

```diff
@@ -18,9 +18,14 @@ class Cgdb < Formula
     depends_on "automake" => :build
   end
 
+  if OS.linux?
+    depends_on "flex" => :build
+  end
   depends_on "help2man" => :build
   depends_on "readline"
 
+  patch :DATA
```

Comment:
To tell the truth I didn't know what is the proper way to fix Linux-related issues. Is it homebrew/homebrew first and then it gets merged to linuxbrew or the other way round. Shall I split this and submit to homebrew/homebrew?